README.md

```diff
@@ -658,8 +664,8 @@ knpu_oauth2_client:
             # a route name you'll create
             redirect_route: connect_drupal_check
             redirect_params: {}
-            base_url: '%env(OAUTH_DRUPAL_BASE_URL)%'
-
+            # Drupal oAuth2 server URL
+            base_url: ''
```

Comment:
I think we can use `->example()` so that the generated README actually shows this `%env(...)%` value, which I think is pretty cool :) https://github.com/knpuniversity/oauth2-client-bundle/commit/60f22367f3eeaf4b5f4576f790ee44f22d2f4827#diff-8ea86beccd406b5a333dd02cbfc2c062R22